PortMaster/pylibs/harbourmaster/hardware.py

```diff
@@ -63,6 +63,7 @@
 
     # Computer/Testing
     'pc':      {'resolution': (640, 480), 'analogsticks': 2, 'cpu': 'unknown', 'capabilities': ['opengl', 'power']},
+    'x86_64':      {'resolution': (1920, 1080), 'analogsticks': 2, 'cpu': 'unknown', 'capabilities': ['opengl', 'power']},
```

Comment:
Change 'cpu' to 'x86_64'.

Reply:
Will do. I will change the resolution to 1024 x 768 to make it more compatible too.

Reply:
Just editing the code now. Should we call this RetroDECK rather than cpu type?

That way can have  JelOS X86_64 as a PC type  too?